### ISSUE TITLE
Firefox Leechblock NG Options Menu: days not selectable

### ISSUE THREAD

**Issue by parslea** [proginosko/LeechBlockNG#128]
![image](https://user-images.githubusercontent.com/59775244/72209064-8d537c00-3478-11ea-90ef-4dd28bb4a8b7.png)

Help! The checkboxes used to be there, disappeared one day. I don't think I updated Firefox, but that could have happened.

**Comment by proginosko** [comment]
Interesting. Which OS are you using?

Try disabling all other extensions and see if the problem is still there.

**Comment by parslea** [comment]
I am using Windows 10 version 1809.

I disabled all other extensions then enabled Leechblock and the boxes returned (thank you!). 

I brought them back one by one and restarted Firefox a few times along the way and the boxes still have not disappeared again, so for now problem solved. I didn't know ostensibly different add-ons can interact with each other in this kind of way, I appreciate your suggestion (and the helpful add-on you created).

If you're curious, the other add-ons I am using are:
LastPass
Video Speed Controller
uBlock Origin
Remove Youtube Suggestions (I believe I added this after onset of the problem)

Thanks again!
